### PR TITLE
doc/new-part-for-the-problem-encountered

### PR DESCRIPTION
**Pull request by mdeport**
update of README for add the problem encountered in detail

### CODE REVIEW

**Comment by mdeport** [comment]
I almost finished the first part.
I only have the second part of the problem left.

File: README.md [hunk 1]
```diff
@@ -1,64 +1,68 @@
...
-    - [How it works](#how-git-flow-works)
-- [GITHUB workflow](#our-github-worflow)
-    - [Issues](#create-an-issue)
-    - [Board](#our-project-board)
-    - [PR](#our-pr)
--[Checks](#check-after-push)
-    - [html5validator](#html5validator)
+  - [Branches](#our-branches)
+  - [How it works](#how-git-flow-works)
+- [GITHUB workflow](#our-github-worflow) - [Issues](#create-an-issue) - [Board](#our-project-board) - [PR](#our-pr) -[Checks](#check-after-push) - [html5validator](#html5validator)
```
Comment (AlexisCro) [inline README.md:9]: Why all of this different point are on the same line now ?? 

The view is not correct now

File: README.md [hunk 2]
```diff
@@ -1,64 +1,68 @@
...
 - Tools
-    - [Commitizen](#commitizen)
+  - [Commitizen](#commitizen)
 - Help / FAQ
-    - [Manage conflicts](#manage-conflicts)
+  - [Manage conflicts](#manage-conflicts)
```
Comment (AlexisCro) [inline README.md:13]: Update summary by adding your topic

File: README.md [hunk 3]
```diff
@@ -241,35 +279,50 @@ So we will create a pull request on GITHUB from main
...
+
+After the release step we deployed our main branch from GITHUB pages.
 
 ### Sprint cycle
-![sprint cycle](./assets/images/img/readme/sprint-cycle.png)
\ No newline at end of file
+
+![sprint cycle](./assets/images/img/readme/sprint-cycle.png)
+
+## Problem encountered
```
Comment (AlexisCro) [inline README.md:323]: Please add this point after the help to resolve conflict it's in the same tropic as help/faq

**Review by AlexisCro — CHANGES_REQUESTED** [review]

**Pull request merged**